pkg/image/image.go

```diff
@@ -392,26 +392,35 @@ func ChangeImageDomain(imageString string, newDomain string) (types.ImageReferen
 	return newImage, nil
 }
 
-func WithoutTag(image string) (string, error) {
-	fullImage, err := AddDefaultRegistry(image, "place.holder.com")
+// GetTag Returns the image tag, the image without the tag, and any errors; in this order.
```

Comment:
It would be good to have some unit tests for this.  I realize that we primarily rely on integration tests for coverage, but this is one of those cases where it is easy to write good unit tests.

Reply:
Added tests